easybuild/easyconfigs/t/TINKER/TINKER-8.11.3-foss-2023a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'TINKER'
+version = '8.11.3'
+
+homepage = 'https://dasher.wustl.edu/tinker'
+description = """The Tinker molecular modeling software is a complete and general package for molecular mechanics
+ and dynamics, with some special features for biopolymers."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://dasher.wustl.edu/tinker/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['8dfbc9fb8f26742d91139187657e2c905744b0243538f81b75bc04cdc2606ff7']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+]
+
+srcdir = 'cmake'
+
+# copy executables into bin directory
+test_cmd = 'find -maxdepth 1 -mindepth 1 -type f -executable |xargs -I % cp % ../tinker/bin/ && '
```

Comment:
It is not clear what this is doing. If it is copying the binaries into `bin` in the installation directory, this needs to be done in the install step or post-install step because the test step can be skipped. If it is copying in a `bin` folder inside the build directory, I fail to see why that copy is needed as that `bin` folder is not in `PATH`.

Reply:
the binaries need to be in the bin directory because their paths are hardcoded in the test scripts.

i have now patched the test scripts instead, which makes it more clear why this is needed, see https://github.com/easybuilders/easybuild-easyconfigs/pull/21098/commits/166f3a68b5e34d20abf63a979ccac7f00a14f966

Reply:
much better, thanks for the fix!